### ISSUE TITLE
Presence of a psuedoelement is hindering with working of links.

### ISSUE THREAD

**Issue by aryankeluskar** [gatsbyjs/gatsby#34380]
### Preliminary Checks

- [X] This issue is not a duplicate. Before opening a new issue, please search existing issues: https://github.com/gatsbyjs/gatsby/issues
- [X] This issue is not a question, feature request, RFC, or anything other than a bug report directly related to Gatsby. Please post those things in GitHub Discussions: https://github.com/gatsbyjs/gatsby/discussions

### Description

the last two elements of the list seem to be blocked by a ::before pseudoelement with a negative margin and height
it was first discovered on Ethereum's website
https://ethereum.org/en/dao/#join-start-a-dao
it is not possible to click the links in the last two list items, due to the aforementioned psuedoelement. .gWTLmC::before and .cHPOmI::before
![image](https://user-images.githubusercontent.com/80093392/147853775-2d32f436-e82f-40d8-a4f9-1d7616b25a2a.png)
Removing it by unticking `content: "" ` under `.gWTLmC::before` in the CSS panel makes the list interactable again.

Original issue in Ethereum repository: https://github.com/ethereum/ethereum-org-website/issues/4930

### Reproduction Link

https://ethereum.org/en/dao/

### Steps to Reproduce

1. Go to https://ethereum.org/en/dao/
2. Scroll to the section in screenshot.
![image](https://user-images.githubusercontent.com/80093392/147853749-f46a3623-2110-4846-9b44-6e7392206c11.png)

3. Try clicking on "Ethereum community DAOs" or "DAOHaus's list of DAOs" or even "Start a colony", ie: last two list items of any list in the subheading of "Join a DAO"
4. You aren't able to click any of them.
5. Since the website uses gatsby for styles, the fix is required in gatsby files.

### Expected Result

Click on the links to open the respective pages mentioned in anchor tag

### Actual Result

Nothing happens, the cursor too stays same rather than turning to a finger.

### Environment

```shell
System:
    OS: Windows 10 10.0.19042
    CPU: (8) ia32 11th Gen Intel(R) Core(TM) i5-1135G7 @ 2.40GHz
  Binaries:
    Node: 14.17.0 - C:\Program Files (x86)\nodejs\node.EXE
    npm: 6.14.13 - C:\Program Files (x86)\nodejs\npm.CMD
  Languages:
    Python: 3.9.5
  Browsers:
    Edge: Spartan (44.19041.1266.0), Chromium (96.0.1054.62)
```

**Comment by marvinjude** [comment]
Hi @aryankeluskar, 

The styles causing this issue seem injected by `styled-components`. Doesn't seem to be an issue with gatsby

Please provide a minimal reproduction link i.e a small gatsby site where you're experiencing this issue

**Comment by LekoArts** [comment]
Hi, please use https://github.com/ethereum/ethereum-org-website/issues/4930 for tracking this issue as this is not related to Gatsby. Thanks!
